.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
     test:
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-18.04
```

Comment:
Can you make sure all workflows use this for now? :)

Reply:
Can we at least use the latest LTS? 20.04?

Reply:
That's what is causing the issues atm; we are going to take a detailed look at this asap, but right now builds would always fail and we need to prevent that